test/doc/developer/reverse.cpp

```diff
@@ -0,0 +1,194 @@
+
+#include <eve/wide.hpp>
+#include <eve/function/bit_cast.hpp>
+#include <eve/function/store.hpp>
+
+#include <algorithm>
+#include <bit>
+#include <concepts>
+
+#include <type_traits>
+
+#if defined(EVE_HW_X86)
+
+namespace eve_extra {
+namespace _reverse {
+
+template <typename T, typename N, typename ABI>
+eve::wide<T, N, ABI> actual_reverse(eve::wide<T, N, ABI> x) requires(sizeof(T) == 8)
```

Comment:
traditionally, we prefer when tests like this `requires` are in fact `if constexpr` inside.

Reply:
I thought about it - but this makes the function 15 miles long. If you want me to - I can do it.

Reply:
As we discussed on Slack, better keep those separated for the clarity of the tutorial.
We'll later do a "you want to write something FOR EVE"  tutorial.